### PR TITLE
[docs] [PLAT-7589] Azure KMS to Key Vault

### PR DESCRIPTION
**Pull request by ddhodge**
Resolves https://yugabyte.atlassian.net/browse/PLAT-7589
Use the correct product name for Azure KMS

@netlify /preview/yugabyte-platform/security/create-kms-config/azure-kms/

### CODE REVIEW

File: docs/content/preview/releases/yba-releases/v2.17.md [hunk 1]
```diff
@@ -564,7 +564,7 @@ docker pull yugabytedb/yugabyte:2.17.0.0-b24
 * [PLAT-6042] Fix kubernetes CPU usage graph
 * [PLAT-6053] [PLAT-6054] Enable toggling force delete for xCluster config
 * [PLAT-6069] Fix: Incremental Backup location same as Full Backup Location
-* [PLAT-6074] Decrypt against all key versions for a key in azure KMS
+* [PLAT-6074] Decrypt against all key versions for a key in Azure Key Vault
```
Comment (aishwarya24) [inline docs/content/preview/releases/yba-releases/v2.17.md:567]: I see a bunch of duplicate entries in the file. It does not relate to your changes , but we have to get rid of the duplicates - Look at lines 143,144, 150,151 and such others. Maybe in another PR.

File: docs/content/stable/yugabyte-platform/security/create-kms-config/azure-kms.md [hunk 2]
```diff
@@ -54,7 +55,7 @@ If you are planning to use an existing cryptographic key with the same name, it
 - Permitted operations should have at least WRAP_KEY and UNWRAP_KEY.
 - The key rotation policy should not be defined in order to avoid automatic rotation.
 
-You can create a KMS configuration that uses Azure KMS, as follows:
+You can create a KMS configuration that uses Azure Key Vault, as follows:
 
 1. Use the YugabyteDB Anywhere UI to navigate to **Configs > Security > Encryption At Rest** to access the list of existing configurations.
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/security/create-kms-config/azure-kms.md:61]: Line 67 - **KMS Provider** — Select **Azure KMS** . This needs to be changed. Present in stable version too

**Review by aishwarya24 — APPROVED** [review]
LGTM.

**Review by NValsan — APPROVED** [review]
Thanks! This turned out to be a larger piece of work than I thought. Thank you for getting this completed so quickly.

**Pull request merged**